Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -69,9 +103,58 @@ public final class ProjectPamphletViewController: UIViewController {
     }
   }
 
+  private func configureViews() {
+    // Configure subviews
+    self.backThisProjectContainerView.addSubview(self.backThisProjectButton)
+
+    self.view.addSubview(self.backThisProjectContainerView)
+    self.view.bringSubviewToFront(backThisProjectContainerView)
```

Comment:
Why necessary to bring to front? Also missing a `self` in this second one.

Reply:
Not necessary, will remove 👍